metadata/src/main/resources/common/metadata/ZkMigrationRecord.json

```diff
@@ -18,7 +18,9 @@
   "type": "metadata",
   "name": "ZkMigrationStateRecord",
   // Version 0 adds ZkMigrationState which is used by the KRaft controller to mark the beginning and end
-  // of the ZK to KRaft migration. Possible values are 1 (PreMigration), 2 (Migration), 3 (PostMigration).
+  // of the ZK to KRaft migration. Possible values are 0 (None), 1 (PreMigration), 2 (Migration), 3 (PostMigration).
```

Comment:
I think this comment needs to be revised to remove the comment about the tagged field, and swap the value of PreMigration and Migration